ultrasonic/src/main/res/values/strings.xml

```diff
@@ -332,6 +334,7 @@
     <string name="settings.wifi_required_title">Download on Wi-Fi only</string>
     <string name="song_details.all">%1$s%2$s</string>
```

Comment:
Please, set this string as **translatable="false"**:
```
<string name="song_details.all" translatable="false">%1$s%2$s</string>
```

Reply:
Ok, done.